compressor/filters/css_default.py

```diff
@@ -6,9 +6,10 @@
 from compressor.conf import settings
 from compressor.filters import FilterBase, FilterError
 
-URL_PATTERN = re.compile(r'url\(([^\)]+)\)')
+URL_PATTERN = re.compile(r'url\( *([\'"]?)(.+?)\1 *\)')
```

Comment:
yeah that's right. the ? makes the + non-greedy. before it said `[^\)]+\)`, that's "take as much as possible but no parenthesis and then a parenthesis, alternatively one can do `.+?\)` which is "take as little as possible before a parenthesis comes".

you right that this should have a comment, i'll see if there is some nice way.